Classes/Bard.md

```diff
@@ -177,4 +177,4 @@ At 6th level, you learn two spells of your choice from any class. A spell you ch
 
 #### Peerless Skill
 
-Starting at 14th level, when you make an ability check, you can expend one use of Bardic Inspiration. Roll a Bardic Inspiration die and add the number rolled to your ability check. You can choose to do so after you roll the die for the ability check, but before the GM tells you whether you succeed or fail.
\ No newline at end of file
+Starting at 14th level, when you make an ability check, you can expend one use of Bardic Inspiration. Roll a Bardic Inspiration die and add the number rolled to your ability check. You can choose to do so after you roll the die for the ability check, but before the GM tells you whether you succeed or fail.
```

Comment:
I have no idea why github thinks these lines are different but they read the same so I don't think it's that big of an issue

Reply:
After poking around I realized it's because originally there wasn't a newline at the end of the file and I think it's better this way